nnpdf_data/nnpdf_data/commondata/ATLAS_TTBAR_13TEV_LJ_DIF/metadata.yaml

```diff
@@ -244,3 +244,121 @@ implemented_observables:
         - - ATLAS_TTBAR_13TEV_LJ_DIF_YTTBAR
         - - ATLAS_TTBAR_13TEV_LJ_DIF_YTTBAR-INTEGRATED
       operation: 'ratio'
+
+  - observable: {description: "double differential cross section w.r.t. transverse momentum and absolute rapidity of top", label: '$\frac{d^2\sigma}{dpT_td|y_t|}$', units: '$\frac{pb}{GeV}$'}
+    observable_name: PTT-YT
+    process_type: HQP_PTQ
+    ndata: 13
+    tables: [649, 650, 651]
+    kinematics:
+      variables:
+        pT_t: {description: "transverse momentum of t", label: "$pT_t$", units: "$GeV$"}
+        y_t: {description: "absolute rapidity of t", label: '$|y_{t}|$', units: ""}
+        m_t2: {description: "top mass squared", label: '$m_{t}^2$', units: '$GeV^2$'}
+        sqrts: {description: "center of mass energy", label: '$\sqrt{s}$', units: 'GeV'}
```

Comment:
Hi @t7phy I think this is good to go. The only comment is that I think it is better to remove the `sqrts`. It just makes thing heavier and the information is in the title of the dataset anyway (and it is accessible by validphys)

Reply:
https://github.com/NNPDF/nnpdf/pull/2218/commits/977b4a17c5bae94a2c1cb8f9202fcc22ada072dd